libs/MobileSync/src/com/salesforce/androidsdk/mobilesync/app/MobileSyncUpgradeManager.kt

```diff
@@ -83,7 +61,29 @@ protected synchronized void upgradeSObject() {
      *
      * @return Currently installed version of mobile sync.
      */
-    public String getInstalledSobjectVersion() {
-        return getInstalledVersion(MOBILE_SYNC_KEY);
+    val installedSobjectVersion: String
+        get() = getInstalledVersion(MOBILE_SYNC_KEY)
+
+    companion object {
+        /**
+         * Key in shared preference file for mobile sync version.
+         */
+        private const val MOBILE_SYNC_KEY = "mobile_sync_version"
+        private const val TAG = "MobileSyncUpgradeManager"
```

Comment:
Is `TAG` unused?  If there are inspector suggestions available that very well checked by the tool, like an unused private member, this might a good time to sweep those away.